opendrift/models/chemicaldrift.py

```diff
@@ -1977,6 +1978,7 @@ def write_netcdf_chemical_density_map(self, filename, pixelsize_m='auto', zlevel
             smoothing_cells:       int, number of cells for horizontal smoothing,
             reader_sea_depth:      string, path of bathimethy .nc file,
```

Comment:
Independent of this PR, but I believe it would be better that this method simply uses the readers added to the simulation object, instead of needing to pass explicit readers/files to this method.